examples/nested_list/src/app.rs

```diff
@@ -48,23 +50,48 @@ impl Component for App {
 
         // note the use of `html_nested!` instead of `html!`.
         let letters = ('A'..='C')
-            .map(|letter| html_nested! { <ListItem name={letter.to_string()} {on_hover} /> });
+            .map(|letter| html_nested! { <ListItem key={format!("letter-{}", letter)} name={letter.to_string()} {on_hover} /> });
 
         html! {
             <div class="main" {onmouseover}>
                 <h1>{ "Nested List Demo" }</h1>
-                <List {on_hover} weak_link={list_link}>
-                    <ListHeader text="Calling all Rusties!" {on_hover} {list_link} />
-                    <ListItem name="Rustin" {on_hover} />
-                    <ListItem hide=true name="Rustaroo" {on_hover} />
-                    <ListItem name="Rustifer" {on_hover}>
-                        <div class="sublist" onmouseover={onmouseoversublist}>{ "Sublist!" }</div>
-                        <List {on_hover} weak_link={sub_list_link}>
-                            <ListHeader text="Sub Rusties!" {on_hover} list_link={sub_list_link}/>
-                            <ListItem hide=true name="Hidden Sub" {on_hover} />
-                            { for letters }
-                        </List>
-                    </ListItem>
+                <List
+                    {on_hover}
+                    weak_link={list_link}
+                    header={
+                        vec![
+                            html_nested! {
+                                <ListHeader text="Calling all Rusties!" {on_hover} {list_link} key="header" />
+                            }
```

Comment:
I would move these `vec!`s out into variables but it's fine if you want to merge it as-is. It's just a stylistic thing to keep Rust code out of `html! `. 3 levels of nested macros doesn't look that either